runtime/moonbase/src/asset_config.rs

```diff
@@ -68,14 +67,9 @@ parameter_types! {
 	pub const MetadataDepositPerByte: Balance = currency::deposit(0, 1);
 }
 
-/// We allow root and Chain council to execute privileged asset operations.
-pub type AssetsForceOrigin = EitherOfDiverse<
-	EnsureRoot<AccountId>,
-	EitherOfDiverse<
-		pallet_collective::EnsureProportionMoreThan<AccountId, CouncilInstance, 1, 2>,
-		governance::custom_origins::GeneralAdmin,
-	>,
->;
+/// We allow Root and General Admin to execute privileged asset operations.
```

Comment:
Did we check with the MBF if it's fine to require at least GeneralAdmin to modify foreign assets?